### PR TITLE
Update sql migration extension strings

### PR DESCRIPTION
**Pull request by rchlkm**
This PR updates sql migration extension strings based on content review, uniforms format to only capitalize first word, adds periods to end of statements.

### CODE REVIEW

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -67,27 +67,27 @@ export function accountLinkedMessage(count: number): string {
...
+export const DATABASE_BACKUP_PAGE_DESCRIPTION = localize('sql.migration.database.page.description', "Select the location of the database backups to use during migration.");
 export const DATABASE_BACKUP_NC_NETWORK_SHARE_RADIO_LABEL = localize('sql.migration.nc.network.share.radio.label', "My database backups are on a network share");
 export const DATABASE_BACKUP_NC_BLOB_STORAGE_RADIO_LABEL = localize('sql.migration.nc.blob.storage.radio.label', "My database backups are in an Azure Storage Blob Container");
 export const DATABASE_BACKUP_NETWORK_SHARE_HEADER_TEXT = localize('sql.migration.network.share.header.text', "Network share details");
-export const DATABASE_BACKUP_NC_NETWORK_SHARE_HELP_TEXT = localize('sql.migration.network.share.help.text', "Provide the network share location that contains backups and the user credentials that has read access to the share");
+export const DATABASE_BACKUP_NC_NETWORK_SHARE_HELP_TEXT = localize('sql.migration.network.share.help.text', "Provide the network share location where the backups are stored, and the user credentials used to access the share.");
```
Comment (aasimkhan30) [inline extensions/sql-migration/src/constants/strings.ts:82]: Do we need a period at the end?

File: extensions/sql-migration/src/constants/strings.ts [hunk 2]
```diff
@@ -233,46 +233,46 @@ export const SUMMARY_MI_TYPE = localize('sql.migration.summary.mi.type', "Azure
...
-export const BLOB_CONTAINER = localize('sql.migration.blob.container.title', "Blob Container");
-export const BLOB_CONTAINER_LAST_BACKUP_FILE = localize('sql.migration.blob.container.last.backup.file.label', "Last Backup File");
+export const SUMMARY_AZURE_STORAGE = localize('sql.migration.summary.azure.storage', "Azure Storage");
+export const NETWORK_SHARE = localize('sql.migration.network.share', "Network share");
+export const BLOB_CONTAINER = localize('sql.migration.blob.container.title', "Blob container");
+export const BLOB_CONTAINER_LAST_BACKUP_FILE = localize('sql.migration.blob.container.last.backup.file.label', "Last backup file");
 export const BLOB_CONTAINER_RESOURCE_GROUP = localize('sql.migration.blob.container.label', "Blob container resource group");
 export const BLOB_CONTAINER_STORAGE_ACCOUNT = localize('sql.migration.blob.container.storage.account.label', "Blob container storage account");
-export const SOURCE_DATABASES = localize('sql.migration.source.databases', "Source Database(s)");
+export const SOURCE_DATABASES = localize('sql.migration.source.databases', "Source database(s)");
```
Comment (aasimkhan30) [inline extensions/sql-migration/src/constants/strings.ts:242]: We should make strings like this plural. I wonder if they localize properly

**Review by aasimkhan30 — APPROVED** [review]

File: extensions/sql-migration/src/constants/strings.ts (hunk 2 above)
Comment (rchlkm) [reply extensions/sql-migration/src/constants/strings.ts]: synced with Kathy. we decided to replace all instances of (s) to the plural noun

File: extensions/sql-migration/src/constants/strings.ts (hunk 1 above)
Comment (rchlkm) [reply extensions/sql-migration/src/constants/strings.ts:82]: the convention seems to have a period at the end of all phrases

**Pull request merged**